packages/common/src/search/_internal/portalSearchUsers.ts

```diff
@@ -71,36 +66,118 @@ export async function portalSearchUsers(
   // so we set it directly
   so.authentication = options.requestOptions.authentication;
 
+  return so as IUserSearchOptions;
+}
+
+/**
+ * @private
+ *
+ * Portal Search Implementation for Users within the currently authenticated user's organization.
+ * Automatically adds "org.name as OrgName" enrichment
+ *
+ * DEPRECATED: This method will be deprecated in a future release, as it's no ideal to impose enrichments for all cases.
+ *
+ * @param query An IQuery object representing the query to serialize
+ * @param options An IHubSearchOptions of search options
+ * @returns a promise that resolves an IHubSearchResponse<IHubSearchResult> of users results
+ */
+export function portalSearchUsersLegacy(
+  query: IQuery,
+  options: IHubSearchOptions
+): Promise<IHubSearchResponse<IHubSearchResult>> {
+  const searchOptions = buildSearchOptions(
+    query,
+    options,
+    "portalSearchUsersLegacy"
+  );
   // Execute search
-  return searchPortal(so as IUserSearchOptions);
+  return searchPortal({
+    ...searchOptions,
+    include: ["org.name as OrgName", ...(searchOptions.include || [])],
+  });
 }
 
 /**
- * Internal portal search, which then converts `IGroup`s to `IHubSearchResult`s
- * handling enrichments & includes along the way
+ * @private
  *
- * @param searchOptions
- * @returns
+ * Portal Search Implementation for Users within the currently authenticated user's organization.
+ * No enrichments added by default.
+ *
+ * @param query An IQuery object representing the query to serialize
+ * @param options An IHubSearchOptions of search options
+ * @returns a promise that resolves an IHubSearchResponse<IHubSearchResult> of users results
  */
-async function searchPortal(
-  searchOptions: IUserSearchOptions
+export function portalSearchUsers(
```

Comment:
Function name - should/can this be `searchPortalUsers()` and `searchCommunityUsers` so it's easier to find (search at beginning) and then `*PortalUsers` is consistent with the entityType